go/private/skylib/lib/paths.bzl

```diff
@@ -65,7 +65,7 @@ def _is_absolute(path):
   Returns:
     `True` if `path` is an absolute path.
   """
-  return path.startswith("/")
+  return path.startswith("/") or path[1] == ":"
```

Comment:
Could cause problems with very short strings or with relative paths that coincidentally have a colon there.

Maybe translate `IsAbs` from $GOROOT/src/path/filepath/path_windows.go (skipping the UNC stuff).

Reply:
Yes, this is far from correct, but it needs to be written properly up stream (bazelbuild/bazel-skylib#9). For now this lets us make progress, and none of the problems are relevant in practice for our only use of this function.